data/human/campaign events.txt

```diff
@@ -1125,7 +1125,7 @@ event "fw abandoned Greenrock"
 		fleet "Large Free Worlds" 9000
 	planet "Greenrock"
 		description `Greenrock was the closest thing that the pirates in southern human space had to a center of government, at least until the Free Worlds occupied it. When rival pirate bands sought to negotiate on neutral territory, this is where they met. Nearly everything was for sale here: illegal weapons, ships customized for pirating, and even slaves.`
-		description `	That was until the Free Worlds occupied the system and cleaned up many of the major population centers. With the Free Worlds gone though, the planet has begun to return to its natural state of crime and villainy.`
+		description `	That was until the Free Worlds occupied the system and cleaned up many of the major population centers. With the Free Worlds no longer able to maintain control, though, the planet has begun to return to its natural state of crime and villainy.`
```

Comment:
This description is already pretty tight as is. Probably just best to keep the description what it was.
```suggestion
		description `	That was until the Free Worlds occupied the system and cleaned up many of the major population centers. With the Free Worlds gone, the planet has begun to return to its natural state of crime and villainy.`
```